app/api/helpers/scheduled_jobs.py

```diff
@@ -142,10 +142,10 @@ def send_event_fee_notification_followup():
                                                         incomplete_invoice.event.id)
 
 
-def expire_pending_tickets_after_three_days():
+def expire_pending_tickets_after_30_min():
     from app import current_app as app
     with app.app_context():
         db.session.query(Order).filter(Order.status == 'pending',
-                                       (Order.created_at + datetime.timedelta(days=3)) <= datetime.datetime.now()).\
+                                       (Order.created_at + datetime.timedelta(minutes=30)) <= datetime.datetime.now()).\
```

Comment:
@mrsaicharan1 @uds5501 one doubt. Order creation is the time when user clicks on `Order Now` button and after that details are filled out. Right?
Then user gets 15 minutes to fill up the order form. So if we see, the user gets only 15 minutes for making the payment. Do you get my point? I think it should be increased to `45 minutes`.

Reply:
@shreyanshdwivedi Good point.